mcodingbot/plugins/python.py

```diff
@@ -42,7 +48,7 @@ async def on_message(event: hikari.MessageCreateEvent) -> None:
         return
 
     pep_links_message = "\n".join(
-        f"PEP {pep_number}: {get_pep_link(pep_number)}"
+        f"PEP {pep_number}: {get_pep_link(pep_number, hide_embed=True)}"
```

Comment:
do you mean, `hide_embed=False`?

Reply:
honestly it would be better to just call it `show_embed` as well

Reply:
No I didn't mean that, the reply can already be annoying, even without the embed.